src/test/functional/features/applicant-middleware.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Homepage
+
+  Background:
```

Comment:
feels like this should be a nightly test

Reply:
that can be arranged